File: Day 113/classwork/cw1.py
# 1) languages = ["Python", "Java", "JavaScript", "C", "C++", "PHP", "Go", "Rust", "Ruby", "Kotlin"] დატოვე მხოლოდ ის ენები რომლების სიგრძეც მეტია 5ზე.

languages = ["Python", "Java", "JavaScript", "C", "C++", "PHP", "Go", "Rust", "Ruby", "Kotlin"]


filtered = list(filter(lambda x: len(x) > 5, languages))

print(filtered)

File: Day 114/classwork/cw1.js
// 1) let languages = ["Python", "JS", "Java", "C++"];
// მოცემულია სიტყვების მასივი. იპოვე სიტყვა “Java”. როგორც კი იპოვი, დაბეჭდე და გააჩერე ლუპი.


let languages = ["Python", "JS", "Java", "C++"];

for(let i = 0;languages.length ; i++){
    if(languages[i] === "Java"){
        console.log("We found Java:")
        break;
    }
}